.github/workflows/R-CMD-check.yaml

```diff
@@ -3,7 +3,7 @@ name: R-CMD-check
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Or do we want to just add `- main` for now, do the branch switch, ensure everything working, and then finally delete the `- main` line?